.travis.yml

```diff
@@ -3,7 +3,8 @@ language: java
 # avoid double builds on pull requests
 branches:
   only:
-    master
+    - master
+    - release/5.0-alpha-2
```

Comment:
You branched from the alpha-2 release branch but the PR is towards master. That doesn't match.

Reply:
I just need to remove this line, is it correct?

Reply:
No, there are more places if you check the diff.

Reply:
I've already changed pom.xml files

Reply:
Ok done